src/ResourceManager/AzureBatch/Commands.Batch/help/New-AzureBatchPool.md

```diff
@@ -76,34 +76,48 @@ The **New-AzureBatchPool** cmdlet creates a pool in the Azure Batch service unde
 
 ## EXAMPLES
 
-### Example 1: Create a new pool using the TargetDedicated parameter set
+### Example 1: Create a new pool using the TargetDedicated parameter set using CloudServiceConfiguration
 ```
-PS C:\>New-AzureBatchPool -Id "MyPool" -VirtualMachineSize "Small" -OSFamily "4" -TargetOSVersion "*" -TargetDedicatedComputeNodes 3 -BatchContext $Context
+PS C:\>$configuration = New-Object -TypeName "Microsoft.Azure.Commands.Batch.Models.PSCloudServiceConfiguration" -ArgumentList @(4,"*")
+PS C:\>New-AzureBatchPool -Id "MyPool" -VirtualMachineSize "Small" -CloudServiceConfiguration $configuration  -TargetDedicatedComputeNodes 3 -BatchContext $Context
+```
+
+### Example 2: Create a new pool using the TargetDedicated parameter set with using VirtualMachineConfiguration
+```
+PS C:\$imageReference = New-Object -TypeName "Microsoft.Azure.Commands.Batch.Models.PSImageReference" -ArgumentList @("UbuntuServer", "Canonical", "16.04.0-LTS", "*")
```

Comment:
Given that this is powershell and we're in a Windows context, we should actually probably use windows 2016 as the example